packages/@aws-cdk/aws-ec2/test/vpc.test.ts

```diff
@@ -588,6 +588,30 @@ describe('vpc', () => {
       });
 
     });
+
+    test('with availabilityZones and maxAzs set, throws error', () => {
+      const stack = getTestStack();
+      expect(() => {
+        new Vpc(stack, 'VPC', {
+          availabilityZones: stack.availabilityZones,
+          maxAzs: 1,
+        });
+      }).toThrow(/Vpc supports either specific 'availabilityZones' or 'maxAzs'/);
+    });
+
+    test('with availabilityZones set', () => {
+      const stack = getTestStack();
+      new Vpc(stack, 'VPC', {
+        availabilityZones: [`${stack.region}b`],
+      });
+      Template.fromStack(stack).resourceCountIs('AWS::EC2::Subnet', 2);
+      for (let i = 1; i < 2; i++) {
+        Template.fromStack(stack).hasResourceProperties('AWS::EC2::Subnet', {
+          AvailabilityZone: `${stack.region}b`,
+        });
+      }
```

Comment:
this loop doesn't do anything, should be:

```suggestion
        Template.fromStack(stack).hasResourceProperties('AWS::EC2::Subnet', {
          AvailabilityZone: `${stack.region}b`,
        });
```

Reply:
Oops, that was a thoughtless copy-paste. Fixed.